stubs/dj-database-url/dj_database_url.pyi

```diff
@@ -0,0 +1,19 @@
+from collections.abc import Mapping
+from typing_extensions import TypedDict
+
+DEFAULT_ENV: str
+SCHEMES: Mapping[str, str]
```

Comment:
Why use `Mapping` here rather than `dict`?

Reply:
Ah, it was the default from create_baseline_stubs. Fixed.

Reply:
Nit for future reference btw: please don't force-push on typeshed PRs. All typeshed PRs are squash-merged at the end of the day, so [it's fine to make a mess](https://github.com/python/typeshed/pull/6560). Also, force-pushing interacts badly with GitHub's UI -- it makes it harder for reviewers to see what changed between commits :)

Reply:
Ah, sorry about that. Habits. It won't happen again. 

Thanks for letting me know.